.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,30 @@
+name: paa-unit-tests-and-quality-workflow
+on:
+  push:
+    branches:
+      - main
+      - feature/**
+  pull_request:
+    branches:
+      - main
+      - feature/**
+jobs:
+  unit-tests-and-quality:
```

Comment:
Is it possible to create 2 distincts jobs, one for unit testing and the other one for quality ?